### PR TITLE
Remove Dev Diagnostics Experimental Feature

### PR DESCRIPTION
**Pull request by krschau**
## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

**Comment by jaholme** [comment]
>       </Extensions>

This should also go as well. It was a part of Dev Diagnostics

---
Refers to: src/Package-Dev.appxmanifest:399 in 0c4e347. [](commit_id = 0c4e347a99223bdd3e28429d81c83f11cae89de7, deletion_comment = False)

**Comment by jaholme** [comment]
>     <ProjectReference Include="..\tools\Utilities\RegPreviewUtility\DevHome.RegistryPreview.csproj" />

this was a piece of devdiagnostics not utilties

---
Refers to: src/DevHome.csproj:108 in 0c4e347. [](commit_id = 0c4e347a99223bdd3e28429d81c83f11cae89de7, deletion_comment = False)

**Comment by dabhattimsft** [comment]
>     <ProjectReference Include="..\tools\Utilities\RegPreviewUtility\DevHome.RegistryPreview.csproj" />

IfeoTool?

---
In reply to: [2430464513](https://github.com/microsoft/devhome/pull/3955#issuecomment-2430464513) [](http://example.com/codeflow?ancestors=2430464513)

---
Refers to: src/DevHome.csproj:108 in 0c4e347. [](commit_id = 0c4e347a99223bdd3e28429d81c83f11cae89de7, deletion_comment = False)

**Comment by jaholme** [comment]
>     <ProjectReference Include="..\tools\Utilities\RegPreviewUtility\DevHome.RegistryPreview.csproj" />

yes ifeotool

---
In reply to: [2430489509](https://github.com/microsoft/devhome/pull/3955#issuecomment-2430489509) [](http://example.com/codeflow?ancestors=2430489509,2430464513)

---
Refers to: src/DevHome.csproj:108 in 0c4e347. [](commit_id = 0c4e347a99223bdd3e28429d81c83f11cae89de7, deletion_comment = False)

**Comment by jaholme** [comment]
>     <ProjectReference Include="..\service\DevHome.Service\DevHome.Service.csproj" />

The NT service should go as well, it was only used by devdiagnostics

---
Refers to: src/DevHome.csproj:87 in 0c4e347. [](commit_id = 0c4e347a99223bdd3e28429d81c83f11cae89de7, deletion_comment = False)

File: src/Package-Can.appxmanifest [hunk 1]
```diff
@@ -346,23 +346,6 @@
         </com2:Extension>
```
Comment (jaholme) [inline src/Package-Can.appxmanifest:346]: These should be removed as well

File: src/Package-Dev.appxmanifest [hunk 2]
```diff
@@ -346,23 +346,6 @@
         </com2:Extension>
```
Comment (jaholme) [inline src/Package-Dev.appxmanifest:346]: Remove this as well, since it was only used by Dev Diagnostcs

**Review by guimafelipe — APPROVED** [review]

**Review by EricJohnson327 — APPROVED** [review]

**Pull request merged**